docs/src/reconciling-local-changes.md

```diff
@@ -0,0 +1,230 @@
+that node will fail, meaning most TPA operations will exit with an error
+and you will be unable to manage the cluster with TPA until you
+reconcile this difference. 
+
+## How to reconcile configuration changes
+In general, the reconciliation process involves modifying `config.yml`
+such that it describes the current state of the cluster and then running
+`tpaexec deploy`.
+
+### Example: parting a PGD node
+Deploy a minimal PGD cluster using the bare architecture and a configure
+command such as:
+
+```bash
+tpaexec configure mycluster \
+-a PGD-Always-ON \
+--platform bare \
+--edbpge 15
+--location-names a \
+--pgd-proxy-routing local
+```
+
+Part a node using this SQL, which can be executed from any node:
+
+  select * from bdr.part_node('node-2');
+
+Rerun `deploy`. Note that, whilst no errors occur, the node is still
+parted. This can be verified using the command `pgd show-nodes` on any
+of the nodes. This is because TPA will not overwrite the metadata which
+tells PGD the node is parted. 
+
+!!! Note 
+It is not possible to reconcile the `config.yml` with this cluster state
+because TPA, and indeed PGD itself, has no mechanism to initiate a node
+in the 'parted' state. In principle you could continue to use TPA to
+continue this parted cluster, but this is not advisable. In most cases
+you will wish to continue to fully remove the node and reconcile
+`config.yaml`.
+!!!
+
+### Example: removing a PGD node completely
+The previous example parted a node from the PGD cluster, but left the
+node itself intact and still managed by TPA in a viable but
+unreconcilable state. 
+
+To completely decommission the node, it is safe to simply turn off the
+server corresponding to `node-2`. If you attempt to run `deploy` at this
+stage, it will fail early when it cannot reach the server. 
+
+To reconcile this change in `config.yml` simply delete the entry under
+`instances` corresponding to `node-2`. It will look something like this:
+
+```yaml
+- Name: node-2
+  public_ip: 44.201.93.236
+  private_ip: 172.31.71.186
+  location: a
+  node: 2
+  role:
+  - bdr
+  - pgd-proxy
+  vars:
+    bdr_child_group: a_subgroup
+    bdr_node_options:
+      route_priority: 100
+```
+
+You can now manage this node as usual using TPA. However, the original
+cluster still has metadata that refers to `node-2` so to complete
+reconciliation it is recommended to run the following SQL on each node
+to remove the metadata. *This step is essential if you wish to add a
+node of the same name in future.*
+
+```sql
+select * from bdr.drop_node('node-2');
+```
+
+!!! Note 
+If you wish to join the original `node-2` back to the cluster after
+removing it in this way, you can do so simply by restoring the deleted
+lines of `config.yml` but you must ensure that `select * from
+bdr.drop_node('node-2');` has been run on this node and that the PGDATA
+directory has been deleted.
+!!!
+
+### Example: changing the superuser password
```

Comment:
Isn't this password management example true for any user declared in config.yml with a TPA generated password?  The superuser password works as described herein regardless, but wouldn't any user declared in config.yml also need their password to be managed in this way?

Reply:
Yes, true. I actually picked this example because I didn't know what would happen. I was curious as to whether TPA used the superuser password to connect at any point and therefore whether changing it might be a blocking change. Turns out it behaves just like other user passwords.